README.md

```diff
@@ -84,8 +84,7 @@ several configurations simultaneously. This helps maintaining packages
 across several versions of OCaml as you can test them all at once
 without hassle.
 
-This feature should make cross-compilation easy, see details in the
-[roadmap](ROADMAP.md).
+To learn more about cross-compilation, read the [documentation](https://dune.readthedocs.io/en/latest/advanced-topics.html#cross-compilation).
```

Comment:
This sentence should probably start with something like: `In particular, this is how Dune handles cross-compilation...`, to make a link with the previous paragraph